Gems/Atom/RHI/Metal/Code/Source/Platform/iOS/RHI/Device_iOS.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#include <RHI/Device_iOS.h>
+#include <Atom_RHI_Metal_Platform.h>
+
+namespace AZ
```

Comment:
I think you can do `namespace AZ::Metal::Platform` here and in other places to flatten the indentation level